.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -75,32 +67,12 @@ jobs:
           skip_ruby: true
           install_playwright: true
           turborepo-server-port: ${{ steps.caches.outputs.port }}
-      - name: generate token
-        id: generate-token
-        uses: tibdex/github-app-token@v1
-        with:
-          app_id: ${{ secrets.GH_BOT_APP_ID }}
-          private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
-      - name: Retrieving coin apps
-        uses: actions/checkout@v4
-        with:
-          ref: generated/ledger-live-bot
-          repository: LedgerHQ/coin-apps
-          token: ${{ steps.generate-token.outputs.token }}
-          path: coin-apps
-      - name: pull docker image
-        run: docker pull ghcr.io/ledgerhq/speculos:sha-e262a0c
-        shell: bash
       - name: Run playwright tests [Linux => xvfb-run]
         id: tests
         run: |
-          export COINAPPS=$PWD/coin-apps
-          if [ "${{ inputs.speculos_tests }}" = true ]; then export MOCK=0 && speculos_tests=":speculos"; fi
-          if [ "${{ inputs.invert_filter }}" = true ]; then invert_filter="-invert"; fi
-          xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- pnpm desktop test:playwright$speculos_tests ${INPUTS_TEST_FILTER:+--grep$invert_filter} "${{ inputs.test_filter }}"
+          xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- pnpm desktop test:playwright ${INPUTS_TEST_FILTER:+--grep} "${{ inputs.test_filter }}"
         env:
           INPUTS_TEST_FILTER: ${{ inputs.test_filter }}
-          SEED: ${{ secrets.SEED_QAA_B2C }}
```

Comment:
I think this is wrongly modified to previous version until this line ?

Reply:
should be good now 🤞 
https://github.com/LedgerHQ/ledger-live/actions/runs/9097085426